codebuild_specs/e2e_workflow_base.yml

```diff
@@ -32,68 +32,41 @@ batch:
       buildspec: codebuild_specs/test.yml
       env:
         compute-type: BUILD_GENERAL1_LARGE
```

Comment:
can you remove these depend-on relationships in the e2e_workflow.yml file as well? so we get the same benefits when running locally